test/primer/classify_test.rb

```diff
@@ -282,11 +282,35 @@ def test_does_not_raise_error_when_passing_in_a_primer_css_class_otherwise
     assert_generated_class("bg-blue text-center float-left ml-1 ",  { classes: "bg-blue text-center float-left ml-1" })
   end
 
+  def test_limits_allocations
+    # Warm up allocations
+    values = { align_self: :center, width: :fit, p: 4, m: 1, border: :top, box_shadow: true, color: :red, visibility: :hidden}
+    Primer::Classify.call(**values)
+
+    assert_allocations 38, within: 4 do
```

Comment:
Where'd this value of `4` come from? Does this value vary a little in local test vs. CI?

Reply:
I should probably add a comment, but I believe the difference is between Ruby and Rails versions, since we run a matrix of versions in CI.

The other thought was to make a build specifically for testing allocations against 2.7, but I thought I'd submit this as-is and get thoughts/feedback first.